README.md

```diff
@@ -42,6 +42,31 @@ If the database schema has changed since last time, you'll need to run
 To see live changes to the frontend while developing, you need to stop the docker
 container and run ```docker-compose up --build``` again.
 
+# Python debugger
+
+To use `pdb`, the Python debugger, follow these steps:
+
+1. Open `docker-compose.yml`. Under the `web` key, insert `stdin_open: true` and `tty: true`. It should look like this:
+
+```yaml
+services:
+  web:
+    stdin_open: true
+    tty: true
+```
```

Comment:
Could we add this permanently? That would make using the debugger a bit easier. I'm not familiar with Docker, so I don't know offhand if that would be OK. I could look into it.

Reply:
Yeah, that's fine IMO